samples/grpc/local-drone-control-scala/src/main/scala/local/drones/DeliveryEvents.scala

```diff
@@ -26,17 +27,14 @@ object DeliveryEvents {
 
     val eventsBySlicesQuery =
       GrpcReadJournal(
+        GrpcQuerySettings(system).withInitialConsumerFilter(
+          // location id already is in the format of a topic filter expression
+          Vector(
+            ConsumerFilter.ExcludeRegexEntityIds(Set(".*")),
+            ConsumerFilter.IncludeTopics(Set(settings.locationId)))),
+        GrpcClientSettings.fromConfig("akka.projection.grpc.consumer.client"),
```

Comment:
Still somewhat inconvenient, we should add another GrpcReadJournal.apply/create overload where you don't have to specify GrpcClientSettings.

Reply:
Actually even worse than I first thought, I'll fix.

Reply:
Fix in https://github.com/akka/akka-projection/pull/998